scripts/main.js

```diff
@@ -0,0 +1,66 @@
+/**
+* Copyright 2017 HUAWEI. All Rights Reserved.
+*
+* SPDX-License-Identifier: Apache-2.0
+*
+*/
+
+
+'use strict';
+
+var path = require('path');
+var fs = require('fs-extra');
+var exec = require('child_process').exec;
+const Util = require('../src/comm/util');
+
+function main() {
+    let program = require('commander');
+    program
+        .option('-c, --config <file>', 'config file of the benchmark')
+        .option('-n, --network <file>', 'config file of the blockchain system under test')
+        .parse(process.argv);
```

Comment:
An `.allowUnknownOption()` should be inserted before the `parse` call, so users can override runtime Caliper settings through the command line (this is made possible by `nconf`)

Reply:
Done :)